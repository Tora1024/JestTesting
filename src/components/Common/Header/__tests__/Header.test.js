import React from 'react';
import { shallow } from 'enzyme';
import ComponentUnderTest from '../Header';

describe('components/Header/Header.js', () => {
  let wrapper;

  describe('in general', () => {
    beforeEach(() => {
      wrapper = shallow(<ComponentUnderTest />);
    });
    it('should render correct snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
