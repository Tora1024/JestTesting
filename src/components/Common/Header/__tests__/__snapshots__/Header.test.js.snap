// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Header/Header.js in general should render correct snapshot 1`] = `
<header
  className="header"
>
  <h1
    className="title"
  >
    A blog in a pond
  </h1>
  <nav
    className="nav"
  >
    <Link
      className="nav-item"
      replace={true}
      to="/"
    >
      Home
    </Link>
    <Link
      className="nav-item"
      replace={true}
      to="/about"
    >
      About
    </Link>
  </nav>
</header>
`;
